src/components/ListView/components/Item/components/Wrapper/Wrapper.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { sizes } from '../../../../../../styles/constants';
+import Hover from '../Hover';
+
+const { KILO, MEGA, GIGA } = sizes;
+const sizeMap = {
+  [KILO]: 'kilo',
+  [MEGA]: 'mega',
+  [GIGA]: 'giga'
+};
+
+const baseStyles = ({ theme }) => css`
+  label: wrapper__item;
+
+  align-items: center;
+  border-radius: ${theme.borderRadius.mega};
+  position: relative;
+  cursor: pointer;
+  &:hover {
+    ${Hover} {
+      display: block;
+    }
+  }
+`;
+
+const paddingStyles = ({ theme, padding }) => css`
+  padding: ${theme.spacings[sizeMap[padding]]};
```

Comment:
I'm only wondering whether the padding on such a list is meant to be responsive?

Reply:
Not sure. Actually in all designs we only use the `GIGA` one. I added some more just in case.